out/main.go

```diff
@@ -4,19 +4,33 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
 	"strconv"
+	"strings"
 
 	"github.com/arbourd/concourse-slack-alert-resource/concourse"
 	"github.com/arbourd/concourse-slack-alert-resource/slack"
 )
 
 func buildMessage(alert Alert, m concourse.BuildMetadata) *slack.Message {
 	fallback := fmt.Sprintf("%s -- %s", fmt.Sprintf("%s: %s/%s/%s", alert.Message, m.PipelineName, m.JobName, m.BuildName), m.URL)
+	log.Printf("MessageFile location passed from yaml: %s", alert.MessageFile)
```

Comment:
Still not sure if this works.

Reply:
As in it wont show? I tried it from the pipeline and the output log showed. I can take it out its not necessary